doc/docs/master/reference/pipeline_spec.md

```diff
@@ -421,6 +425,18 @@ nothing to process. For more information about scheduling GPUs see the
 [Kubernetes docs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/)
 on the subject.
 
+### Sidecar Resource Limits (optional)
+
+`sidecar_resource_limits` determines the upper threshold of resources
+allocated to the sidecar, init, or storage containers. You can also
+set the init container limit to the user container limit. 
```

Comment:
Setting the init container limits to the user container limits is automatic - there is no user control over this.

Reply:
corrected.